recipes/fastjet/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "fastjet" %}
+{% set version = "3.3.4.0rc8" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fastjet-{{ version }}.tar.gz
+  sha256: 30ffebf47389c7e2d8b40a6e863175848fe59c8afc5e276612b7ccf3b2a33ed5
+
+build:
+  number: 0
+  skip: true  # [not linux]
```

Comment:
@jpivarski  the Windows and macOS builds are just processing the skip and passing.

Reply:
@jpivarski yes - sorry I turned off those builds to keep things simple for now.